jupyter-extension/scripts/jhub-dev-helm-values.yml

```diff
@@ -0,0 +1,46 @@
+# Note: These values are for local testing only, not Production
+singleuser: # singleuser refers to the jupyter notebook that gets created by Hub
+  cloudMetadata:
+    blockWithIptables: false # JH modifies IP tables on the kubernetes hosts to block access to cloud metadata
+  defaultUrl: '/lab' # Ensure we're launching Jupyterlab, rather than jupyter notebooks
+  image:
+    name: pachyderm/notebooks-user # In sidecar mode the user can use their own image (need to install the jupyter extension
```

Comment:
adding instructions for building the image might be helpful

Reply:
I have a note in Jira for that as a future task